File: scala/src/test/scala/com/gildedrose/GildedRoseTest.scala
package com.gildedrose

import org.scalatest.flatspec.AnyFlatSpec
import org.scalatest.matchers.should.Matchers

class GildedRoseTest  extends AnyFlatSpec with Matchers {
  // TODO : Ignored
  it should "foo" ignore {
    val items = Array[Item](new Item("foo", 0, 0))
    val app = new GildedRose(items)
    app.updateQuality()
    app.items(0).name should equal ("fixme")
  }
}

File: csharp.NUnit/GildedRoseTests/GildedRoseTest.cs
using System.Collections.Generic;
using GildedRoseKata;
using NUnit.Framework;

namespace GildedRoseTests;

public class GildedRoseTest
{
    [Test]
    [Ignore("WIP")]
    public void Foo()
    {
        var items = new List<Item> { ItemFactory.CreateItem("foo", 0,0) };
        var app = new GildedRose(items);
        app.UpdateQuality();
        Assert.That(items[0].Name, Is.EqualTo("fixme"));
    }
}
